.yetus/excludes.txt

```diff
@@ -1,12 +1,15 @@
 ^tests/integration/config.yml
+^tests/eden/eclient/testdata/maridb.txt
 ^pkg/mkimage-raw-efi/initramfs-init
 ^pkg/mkimage-raw-efi/nlplug-findfs.c
 ^pkg/pillar/rootfs/fscrypt.conf
 ^pkg/fw/
 ^pkg/debug/abuild/
 ^pkg/pillar/zedpac/
+^pkg/kvm-tools/
 ^api/python/
 ^api/go/
+^api/proto/
```

Comment:
My exclude this? We want checks on the proto files so we can incrementally improve things and make new defintions better. But we are not doing changes to existing definitions.

Reply:
I've excluded because we are not checking the output (api/go)... I will enable the checks again...